app/src/main/java/com/venuenext/venuenextsdkdemo/MainActivity.kt

```diff
@@ -56,6 +56,34 @@ class MainActivity : AppCompatActivity(), BottomNavigationView.OnNavigationItemS
         // Setting a reselected listener will disallow selection of the same tab
         // bottomNavigationView.setOnNavigationItemReselectedListener { /* Disallow reselection */ }
 
+        // This is an example of how to initialize the SDK if you have reveived a JSON config file
+        //from VenueNext.
+//        val sdkKey = "YOUR_VN_SDK_KEY"
+//        val sdkSecret = "YOUR_VN_SDK_SECRET"
+//
+//        val configStream = resources.openRawResource(
+//            resources.getIdentifier("vn_sdk_config", "raw", packageName)
+//        )
+//        var configString = ""
+//        try {
+//            configStream.bufferedReader().use { configString = it.readText() }
+//        }
+//        catch (e: Exception) {
+//            Log.e(TAG, "Error reading the VN SDK config file", e)
+//        }
+//
+//        VenueNext.initialize(
+//            sdkKey = sdkKey,
+//            sdkSecret = sdkSecret,
+//            context = this,
+//            javaWebToken = null,
+//            configJsonString = configString,
+//            onSuccess = this::completeInitialize,
+//            onError = this::completeInitializeAfterFailure
+//        )
+
+        // This is an example of how to initialize the SDK if you have NOT received a JSON config
+        //file from VenueNext.
```

Comment:
Maybe just a slight change to the language here. Something like "This is an example of initializing the SDK without a JSON config file. This method of initialization is currently deprecated and will be removed in a future release." Thoughts?

Reply:
I'm good with that. I like it. I'll make the change.

Reply:
Lol. Gota make new stashes. At least with the config file the stash is just 1 file.

Reply:
Two I guess, will have to change the SDK key and secret in MainActivity as well.